desktops/kde.txt

```diff
@@ -0,0 +1,434 @@
+KDE
+________________________________________________________________________________
+
+KDE [0] is desktop environment which consists in the Plasma Desktop, the KDE 
+Frameworks, and a collection of free and open source, cross-plastform  programs.
```

Comment:
`cross-plastform` -> `cross-platform`

Two spaces between above and "programs".

Reply:
Fixed!